behind-the-veil-siteroot/SettingUp.md

```diff
@@ -59,7 +59,10 @@ Once all your dependencies have been installed:
   meteor npm start
   ```
   Your app should automatially open on the browser. If it doesn't, the console will show you a link that the app is running on the browser from so click this.
-
+## Testing
+```console
+meteor test --driver-package meteortesting:mocha 
```

Comment:
If you want, we can also let everyone know that you can also run `meteor npm test` instead (might be easier to remember), since we have this defined as a script in our package.json

Reply:
Heres the weird thing, none of my tests show command line, but when u do run with the driver package and open up local host it shows all the tests.